docs/docs/integrations/providers/pebblo/index.md

```diff
@@ -0,0 +1,20 @@
+# Pebblo
+
+[Pebblo](https://www.daxa.ai/pebblo) enables developers to safely load and retrieve data to promote their Gen AI app to deployment without
+worrying about the organization’s compliance and security requirements. The Pebblo SafeLoader identifies semantic topics and entities found in the
+loaded data and the Pebblo SafeRetriever enforces identity and semantic controls on the retrieved context. The results are
+summarized on the UI or a PDF report.
+
+
+## Pebblo Overview:
+
+`Pebblo` provides a safe way to load and retrieve data for Gen AI applications. 
+It includes:
+1. **Identity-aware Safe Loader** that loads data and identifies semantic topics and entities.
+2. **SafeRetrieval**  that enforces identity and semantic controls on the retrieved context.
+3. **User Data Report** that summarizes the data loaded and retrieved.
+
+## Example Notebooks
+
+For a more detailed examples of using Pebblo, see the following notebooks:
+* [this notebook](/docs/integrations/providers/pebblo/pebblo_retrieval_qa.ipynb) shows how to use Pebblo retrieval QA to safely retrieve data.
```

Comment:
Instead of _this notebook_ use a bit more appropriate text
```suggestion
* [PebbloRetrievalQA](/docs/integrations/providers/pebblo/pebblo_retrieval_qa.ipynb) shows how to use Pebblo retrieval QA to safely retrieve data.
```